src/lib/controller/index.ts

```diff
@@ -322,3 +308,26 @@ export class Controller {
     }
   }
 }
+
+export function getContextAndLog(
+  req: express.Request,
+  admissionKind: "Mutate" | "Validate",
```

Comment:
I think we have a defined type enum that represents this union type of two strings? Maybe we can use that here instead? Check one of the `types.ts` files.